README.md

```diff
@@ -60,6 +60,8 @@ keeping it up to date for you.
 - [Honeycomb.io](https://github.com/honeycombio/opentelemetry-demo)
 - [Lightstep](https://github.com/lightstep/opentelemetry-demo)
 - [New Relic](https://github.com/newrelic/opentelemetry-demo)
+- [Splunk](https://github.com/signalfx/opentelemetry-demo)
+- [Sumo Logic](https://github.com/SumoLogic/opentelemetry-demo)
```

Comment:
This seems like changes from the upstream main, could we exclude them from this PR please?

Reply:
ok